docker/dist/Dockerfile.dist.centos7

```diff
@@ -1,4 +1,3 @@
-#  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This line was deleted by mistake.

Reply:
I fixed the error and updated my branch. Thank you!

Reply:
> I fixed the error and updated my branch. Thank you!

thx~